server/lib/job-queue/handlers/video-channel-import.ts

```diff
@@ -0,0 +1,41 @@
+import { logger } from '@server/helpers/logger'
+import { YoutubeDLCLI } from '@server/helpers/youtube-dl'
+import { CONFIG } from '@server/initializers/config'
+import { VideoChannelModel } from '@server/models/video/video-channel'
+import { VideoChannelImportPayload } from '@shared/models'
+import { Job } from 'bull'
+import { synchronizeChannel } from '@server/lib/video-import-channel'
+
+export async function processVideoChannelImport (job: Job) {
+  const payload = job.data as VideoChannelImportPayload
```

Comment:
I wonder if there could not be some issues with video orders when:
- the user creates a synchronization, requesting a channel import
- the channel import runs while a sync is run

I wonder if we should no passe an optional `syncId` in the payload to flag the synchronization as being processed, and skip any of sync in `PROCESSING`.

What do you think @Chocobozzz?

Reply:
Maybe we can use the same queue for these two tasks so they are processed sequentially.
I'll check taht

Reply:
I'll check how we can handle this case and update the code